MyHeartCounts/MyHeartCountsStandard+Preview.swift

```diff
@@ -0,0 +1,26 @@
+//
+// This source file is part of the My Heart Counts iOS application based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import Spezi
+import SpeziAccount
+import SpeziStudy
+
+
+extension MyHeartCountsStandard {
+    @MainActor @ModuleBuilder static var previewModels: ModuleCollection {
```

Comment:
might make sense to have this be an extension on the `MyHeartCounts` struct, rather than the Standard?